ios/FluentUI/Core/Theme/Tokens/TokenResolver.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// Determines which tokens to use, and ensures they refer to the correct `FluentTheme` instance.
+func resolvedTokens<T: TokenizedControlInternal>(for control: T, fluentTheme: FluentTheme) -> T.TokenType {
+    let tokens = control.state.overrideTokens ?? fluentTheme.tokens(for: control) ?? control.state.defaultTokens
```

Comment:
Just for confirmation: The sample of how to set a custom ControlToken for a specific theme (`fluentTheme.tokens(for: control)`) will come on a future PR?

Reply:
Yep! That will be part of addressing the testability task.